android/app/build.gradle

```diff
@@ -135,8 +135,8 @@ android {
         applicationId "me.rainbow"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 234
-        versionName "1.9.42"
+        versionCode 236
```

Comment:
had to bump one more as I had to upload twice to Google playstore as I accidentally pressed back button on browser and had to re-upload.